### PR TITLE
[UiBundle] Unification os services name

### PR DESCRIPTION
**Pull request by Wojdylak**
| Q               | A
|-----------------|-----
| Branch?         | 2.0
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | #17181
| License         | MIT

### CODE REVIEW

File: UPGRADE-2.0.md [hunk 1]
```diff
@@ -144,6 +144,9 @@
 | `sylius.tax_rate_date_eligibility_checker`                                                                 | `sylius.eligibility_checker.tax_rate_date`                                           |
 | **TaxonomyBundle**                                                                                         |                                                                                      |
 | `sylius.doctrine.odm.mongodb.unitOfWork`                                                                   | `sylius.doctrine.odm.mongodb.unit_of_work`                                           |
+| **UiBundle**                                                                                               |                                                                                      |
+| `Sylius\Bundle\UiBundle\Twig\RedirectPathExtension`                                                        | `sylius.twig.extension.redirect_path`                                                |
+| `Sylius\Bundle\UiBundle\Storage\FilterStorage`                                                             | `sylius.twig.storage.filter`                                                         |
```
Comment (GSadee) [inline UPGRADE-2.0.md]:
```suggestion
```

File: src/Sylius/Bundle/UiBundle/Resources/config/services/twig.xml [hunk 2]
```diff
@@ -23,32 +23,18 @@
...
-        <service id="Sylius\Bundle\UiBundle\Twig\RedirectPathExtension">
+        <service id="sylius.twig.extension.redirect_path" class="Sylius\Bundle\UiBundle\Twig\RedirectPathExtension">
             <argument type="service" id="sylius.twig.storage.filter" />
             <argument type="service" id="router" />
             <tag name="twig.extension" />
         </service>
-        <service id="sylius.twig.extension.redirect_path" alias="Sylius\Bundle\UiBundle\Twig\RedirectPathExtension" />
 
-        <service id="Sylius\Bundle\UiBundle\Storage\FilterStorage" public="false">
+        <service id="sylius.twig.storage.filter" class="Sylius\Bundle\UiBundle\Storage\FilterStorage" public="false">
```
Comment (GSadee) [inline src/Sylius/Bundle/UiBundle/Resources/config/services/twig.xml]: This one should be also removed

File: src/Sylius/Bundle/UiBundle/templates/security/_login.html.twig
Comment (GSadee) [inline src/Sylius/Bundle/UiBundle/templates/security/_login.html.twig:1]: Do we still need this template? Is it used anywhere?

Comment (Wojdylak) [reply src/Sylius/Bundle/UiBundle/templates/security/_login.html.twig:1]: Yes, it is used in the `templates/security/login.html.twig` file, and this template is utilized by the `Sylius\Bundle\UiBundle\Controller\SecurityController`.

**Review by GSadee — APPROVED** [review]

**Pull request merged**